iOSLearningProject/iOSLearningProject/ExtensionsForCustomization.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ExtensionsForCustomization.swift
+//  iOSLearningProject
+//
+//  Created by Srushti Sarju Engineer on 25/02/25.
+//
+
+import Foundation
+import UIKit
+
+extension UIButton {
+    func applyGradient() {
```

Comment:
As pointed out previously this is not the usecase for the extensions, extension properties and functions should be general. This is applying specific gradient